dask/dataframe/core.py

```diff
@@ -3134,6 +3165,62 @@ def memory_usage(self, index=True, deep=False):
         )
         return delayed(sum)(result.to_delayed())
 
+    @derived_from(pd.Series)
+    def agg(self, arg):
+        numeric_only_aggs = ["prod", "var", "std", "mean"]
+        other_aggs = ["any", "sum", "min", "max", "count"]  # size
+        supported_funcs = numeric_only_aggs + other_aggs
+
+        if isinstance(arg, str):
+            if arg in supported_funcs:
+                return getattr(self, arg)()
+            else:
+                raise ValueError(
+                    "unknown/unsupported aggregate function: {}".format(arg)
+                )
+        elif isinstance(arg, Iterable):
+            supported_aggs = [argfunc in supported_funcs for argfunc in arg]
+            if all(supported_aggs):
+                from .multi import concat
+
+                series = []
+                for aggfunc in arg:
+                    if aggfunc == "size":
+                        series.append(
+                            getattr(
+                                self.to_frame().rename(None, {self.name: aggfunc}),
+                                aggfunc,
+                            )
+                        )
+                    else:
+                        # dask.DataFrame.prod returns a value, even for object dtype
+                        if not (
+                            aggfunc in numeric_only_aggs and self.dtype == "object"
+                        ):
+                            series.append(
+                                getattr(
+                                    self.to_frame().rename(None, {self.name: aggfunc}),
+                                    aggfunc,
+                                )()
+                            )
+                return concat(series).repartition(npartitions=1)
```

Comment:
Instead of going through conversion to frames and concatenation, I might do the following instead (untested):

```python
values = []
for aggfunc in arg:
    # Accumulate values for aggs in order
    if aggfunc == "size":
        val = self.size
    else:
        val = getattr(self, aggfunc)()
    values.append(val)

name = "agg-combine-" + tokenize(values)
dsk = {(name, 0): (agg_combine, [v.key for v in values], arg, self.name)}
meta = self._meta.agg(arg)
graph = HighLevelGraph.from_collections(name, dsk, dependencies=values)
return new_dd_object(graph, name, meta, divisions=(None, None))
```

where 

```python
def agg_combine(values, index, name):
    return pd.Series(values, index=index, name=name)
```

This avoids converting to a dataframe and renaming, and should work fine with `.size`.